marthas_dashboard/analysis/Association_Rules_Orange.py

```diff
@@ -26,10 +26,7 @@ def getDFBuildingValuesInRange(building_id, start_date_time, stop_date_time):
     # data = api.building_values_in_range('4', '2016-08-18', '2017-08-19')
     df = api.building_values_in_range(building_id, start_date_time, stop_date_time)
 
-    df_pivot = (df
-              .groupby(['pointtimestamp', 'pointname'])['pointvalue']
-              .sum().unstack().reset_index().fillna(0)
-              .set_index('pointtimestamp'))
+    df_pivot = df.pivot_table(values='pointvalue', index='pointtimestamp', columns='pointname')
```

Comment:
Yes, I did see this! The reason that it got changed to this sort of pivoting was for something in the other algorithm that we were trying (the apriori example), but if this works for Orange, It's definitely cleaner!

Reply:
Gotcha! As far as I can tell, the two lines are equivalent in this context:

*OLD:*
![screen shot 2018-01-23 at 1 42 40 pm](https://user-images.githubusercontent.com/16504363/35297009-547bf8b4-0043-11e8-924c-a83c85eca9f7.png)

*NEW:*
![screen shot 2018-01-23 at 1 40 11 pm](https://user-images.githubusercontent.com/16504363/35296953-2f2211f2-0043-11e8-8b3e-24cc237cba27.png)

But maybe you needed slightly more control in the apriori context?

Reply:
And again, it doesn't really matter. I just think its fun to exchange Pandas tricks!